### DISCUSSION TITLE
0.27 time is now synced to UTC instead of my local time

### DISCUSSION THREAD

**Discussion by bobsbbq** [gdyuldin/x6100_gui#38] [General]
0.27 I noticed the time is now UTC instead of my local time. I don't know if there is a way to change it. I just noticed this on the latest version last night. I had not noticed this before. N5BSB

**Comment by gdyuldin** [comment]
It's by design. ADIF log should use UTC time, also it's helpful for SSB/CW during POTA - to write correct time to your paper log.

**Reply by bobsbbq** [reply]
Ok, that makes sense. Thank you.
